zephyr/esp_shared/include/stubs.h

```diff
@@ -24,7 +24,7 @@
 #define DT_CPU_COMPAT espressif_riscv
 #endif
 
-#define ESP_SOC_DEFAULT_CPU_FREQ_MHZ ((DT_PROP(DT_INST(0, DT_CPU_COMPAT), clock_frequency)))
+#define ESP_SOC_DEFAULT_CPU_FREQ_MHZ ((DT_PROP(DT_INST(0, DT_CPU_COMPAT), clock_frequency)) / 1000000)
```

Comment:
```suggestion
#define ESP_SOC_DEFAULT_CPU_FREQ_MHZ MHZ(DT_PROP(DT_INST(0, DT_CPU_COMPAT), clock_frequency))
```

Reply:
`clock_frequency` is using HZ units, defined in `esp32_clock.h`